ush/templates/FV3LAM_wflow.xml

```diff
@@ -284,12 +290,11 @@ MODULES_RUN_TASK_FP script.
     <envar><name>ICS_OR_LBCS</name><value>LBCS</value></envar>
 
     <dependency>
-      <!-- Files through fhr 39 exist. -->
       <sh>
-        test $(find <cyclestr offset="-{{ extrn_mdl_lbcs_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_lbcs }}/@Y@m@d@H/postprd/</cyclestr> \
-               -name wrfnat_rr_[0-3][0-9].grib2 -mmin +5 | wc -l) \
-        == 40
-      </sh>
```

Comment:
The single find is meant to reduce the stress on the file system. It would be best to use as few `find` commands as possible to help with speedy `rocotorun` execution.

Reply:
Good point. I know as much about "find" syntax as I do about Ancient Greek.  I'll brush up and see if I can make this a one-liner

Reply:
Latest commit has this reduced to a single "find" command

Reply:
@TrevorAlcott-NOAA Would you mind posting the resulting dependency in a comment here so we can see what the result of this template is? Or pointing us to a complete FV3LAM_wflow.xml on disk?

Reply:
test $(find <cyclestr offset="-3:00:00">/misc/whome/rtrr/rap/@Y@m@d@H/postprd/</cyclestr>\
        \( -name wrfnat_rr_03.grib2 -mmin +5 \
        -o -name wrfnat_rr_06.grib2 -mmin +5 \
        -o -name wrfnat_rr_09.grib2 -mmin +5 \
        -o -name wrfnat_rr_12.grib2 -mmin +5 \
        -o -name wrfnat_rr_15.grib2 -mmin +5 \
        \) | wc -l ) == 5

Reply:
Thanks! What does `-o` do?

Reply:
It means "or".  Intuitively it would make more sense to just use "-a" ("and") and avoid the counting at the end, but the way "find" interprets "and" is "does a single file have both of these names", not "does any file have this name or that name", and the former is of course impossible because a file can't have two different names simultaneously

Reply:
Thanks! That's helpful. :)